recipes/gmp/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "6.1.0" %}
+
+package:
+  name: gmp
+  version: {{ version }}
+
+source:
+  fn: gmp-{{ version }}.tar.bz2
+  url: ftp://ftp.gnu.org/gnu/gmp/gmp-{{ version }}.tar.bz2
+  md5: 86ee6e54ebfc4a90b643a65e402c4048
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - m4
+
+test:
+  files:
+    - test.c
+
+about:
+  home: http://gmplib.org/
+  license: GPL 2 and LGPL 3
+  summary: The GNU multiprecision library.
+
+extra:
+  recipe-maintainers:
+    - asmeurer
+    - isuruf
+    - jakirkham
```

Comment:
@asmeurer @isuruf, I went ahead and added both of you as recipe maintainers. This will give you commit privileges on the gmp feedstock (repo) once generated. This helps with changing the recipe used to build and updating the version to generate new releases as they become available. Is this ok?